src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -7,6 +7,16 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        String answer = "";
```

Comment:
you have some problem with tests. try to fix it
![image](https://user-images.githubusercontent.com/36365171/70030061-17264480-15b1-11ea-9f34-d5b37e90fcb8.png)